Composer/packages/client/src/pages/botProject/BotProjectSettings.tsx

```diff
@@ -119,6 +121,7 @@ const BotProjectSettings: React.FC<RouteComponentProps<{ projectId: string; skil
     <Page
       useDebugPane
       data-testid="BotProjectsSettings"
+      getStarted={newCreationFlowFlag}
```

Comment:
Instead of drilling this prop I'd prefer if the Page component pulled in the flag and handled the conditional rendering transparently from the caller here unless I'm missing a separate motivation

Reply:
Eventually, this will just be getStarted={true}, but for now, we only want this component to appear when the flag is turned on.

If we check inside the component, we have to do the additional work of testing if the page in question is the settings page.

Reply:
Got it. Can we change the prop name to `useGettingStarted` to align with `useDebugPane` so it can layer like:

```jsx
<Page
  useDebugPane
  useGettingStarted
  ...
/>
```

Reply:
done